app/views/users/show.html.erb

```diff
@@ -1 +1,49 @@
-<h1>User's page</h1>
\ No newline at end of file
+<%= stylesheet_link_tag ('home') %>
+<section class="users-home">
+  <div class="user-main">
+    <div class="photo">
+      photo
+    </div>
+    <div class="name">
+      <ul>
+        <li>
+          Username: <%= @user.name %>
+        </li>
+      </ul>
+        <p class="user-activity">
+          <%= link_to "view post", user_path(id: @user.id) %>
+        </p>
+    </div>
+  </div>
+  <div class="bio">
+    <div class="bio-body">
+      <h5>Bio</h5>
+      <p>
+        <%= @user.bio %>
+      </p>
+    </div>
+  </div>
+  
+  <div class="post">
+    <% @user.posts.each_with_index do |post, index| %>
```

Comment:
- [ ] Kindly consider using the `recent_posts` to render the last 3 posts of the user, as for now your returning all the posts